git_buildpackage

```diff
@@ -163,9 +172,27 @@ def create_source_package(repo_dir, output_dir, revision, build_args):
     safe_run(command, cwd=repo_dir, interactive=True)
 
 
+    # Lets find the .dsc
+    #if not os.path.exists(
+    found_dsc = False
+    for fname in os.listdir(absoutput_dir):
+        if fnmatch.fnmatch(fname, '*.dsc'):
+            input_file = os.path.join(absoutput_dir, fname)
+            found_dsc = True
+            with open(input_file, 'a') as dsc_file:
+                dsc_file.write('Git-Id: ' + git_id)
+                dsc_file.write('Git-URL: ' + url)
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Daniel,

Would you be happy if this was default enabled, with option to disable the URL? Would you like this for the git-id also? I'm guessing this is a privacy concern?

Cheers
Colin.

On 21 April 2016 06:33:02 BST, Daniel Gollub <notifications@github.com> wrote:
>> @@ -163,9 +172,27 @@ def create_source_package(repo_dir, output_dir,
>revision, build_args):
>>      safe_run(command, cwd=repo_dir, interactive=True)
>>
>>
>> +    # Lets find the .dsc
>> +    #if not os.path.exists(
>> +    found_dsc = False
>> +    for fname in os.listdir(absoutput_dir):
>> +        if fnmatch.fnmatch(fname, '*.dsc'):
>> +            input_file = os.path.join(absoutput_dir, fname)
>> +            found_dsc = True
>> +            with open(input_file, 'a') as dsc_file:
>> +                dsc_file.write('Git-Id: ' + git_id)
>> +                dsc_file.write('Git-URL: ' + url)
>
>Could you make the exposing of the Git-URL configurable?
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/gollub/obs-service-git_buildpackage/pull/1/files/dcaaaf13ae8364b587058c1b4e7ffc17e55de7bd#r60528145
-----BEGIN PGP SIGNATURE-----
Version: APG v1.1.1

iQJABAEBCgAqBQJXGHReIxxDb2xpbiBEb2NoZXJ0eSA8Y29saW5AZG9jaC5vcmcu
dWs+AAoJEPNJPE2Awx82p/oQALWMWJa2hB5dkQSvB1isvALqjt2KpNDHGhlntc1k
mie/1csR6k+mT/kN+EttmxsAzG4p+as6jleneRx1BQxuN6UfAx3G3oHLhWphuZDX
QNOrFZhD9Y9rsdc4jX0tY1PhSJPmQ4s2y3N4dQFlovX0f0tFiLhXF6zBIlPWDUKh
U62ffXc4LfCaVcjwPD+cYqvnkQDThHtiKSsW5aCXy1ccnAGyU6VjaC7g89h3hxQC
QdHSgAHn3LKB0E4X7tE6DdozvHZCb+KyP40l6xc9yfQD6eNw4q5bGdAU2DH4CT8U
5QfUpvKEoxNkTUoJ4DKpqSzkrglieDAZ1eVDgJXc4+tPYJn5pA+6iOiIjEQ71sXC
YUa0kP8sJdRwoH9Svz8mXeqfNuWMIJ4KHq60hau14WLS6mbJjoEq0DJdTV0GrT70
6leeKvNVtvjgCK56Zw3RhiChgy9V+BQ+PQ6AK6j4d4cRXhtyWZ3OajZum6AnqMCs
7TedDTdXWwuk3++W+zg/Ub9Fewp/WtnMSuN5ri+quPVBrJpf3UQ+w9EBPgYdA0+y
jKcYJSJorGPOc3fx7coQ8tPK0d7JjnkXksHHUUxQBaZyVtUPwa1W9DT3iEEE2Ffo
dM00MruS6KOH+/IK/zFLj20O/Ha3EsHZ/JgriRFzU5x/muwcVqE1bhtKUlNdwoT/
GnrS
=MmsW
-----END PGP SIGNATURE-----